src/Platforms/AbstractPlatform.php

```diff
@@ -190,6 +193,23 @@ public function getBinaryTypeDeclarationSQL(array $column): string
         }
     }
 
+    /**
+     * Returns the SQL snippet to declare an ENUM column.
+     *
+     * Enum is a non-standard type that is espacially popular in MySQL and MariaDB. By default, this method throws
+     * because we assume that the platform does not support ENUM types.
```

Comment:
this does not correspond to the implementation

Reply:
Yes, I forgot to update this part. Should be fixed now.